apps/prairielearn/src/lib/regrading.sql

```diff
@@ -13,7 +13,7 @@ FROM
   JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
   JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
   JOIN pl_courses AS c ON (c.id = ci.course_id)
-  LEFT JOIN users AS u USING (user_id)
+  LEFT JOIN users AS u ON (u.user_id = ai.user_id)
```

Comment:
While I was here, I changed this to the more explicit form, which I think we prefer?

Reply:
Yep, we definitely prefer the explicit ON. Someday we might even manage to rename `users.user_id` to `users.id` 😂